web-api/src/ingestion/routes.rs

```diff
@@ -79,6 +79,11 @@ pub(super) fn configure_service(config: &mut ServiceConfig) {
                 .route(web::get().to(get_document_candidates))
                 .route(web::put().to(set_document_candidates)),
         )
+        .service(
+            web::resource("/documents/_indexed_properties")
+                .route(web::post().to(create_indexed_properties))
+                .route(web::get().to(get_indexed_properties_schema)),
```

Comment:
good point I will add it, because we where rushing things a bit I wanted to make sure the code which potentially needs more review fixes is done first